Wikipedia/Code/MWKDataStore.m

```diff
@@ -114,8 +114,9 @@ - (instancetype)initWithContainerURL:(NSURL *)containerURL {
 }
 
 - (void)teardown:(nullable dispatch_block_t)completion {
+    [self stopCoreDataSynchronizers];
+    [self.session teardown];
```

Comment:
Teardown the core data synchronizers, ensure session teardown is called even if there's no cache controller.